factory/20.04/stretch_create_ament_workspace.sh

```diff
@@ -7,12 +7,13 @@ echo "###########################################"
 echo "CREATING GALACTIC AMENT WORKSPACE at $AMENT_WSDIR"
 echo "###########################################"
 
+source /opt/ros/galactic/setup.bash
```

Comment:
@hello-fazil We actually don't want to source this before the if statement. `$ROS_DISTRO && ! $ROS_DISTRO = "galactic"` checks whether any other version of ROS was sourced beforehand. If we source galactic before the check, it won't be able to tell if some other version was sourced beforehand.

It's important to know if two conflicting versions of ROS are sourced because it creates issues with rosdep later on. Can you revert this commit?

Reply:
I got It, Thanks for catching this. I will revert this commit! 
I misunderstood the context of this script, thinking the if statement is to verify if the ROS_DISTRO environment variable exists and it is set to galactic. 
While running the `stretch_new_user_install.sh` script which first runs the script `stretch_create_catkin_workspace.sh` that creates the `catkin_ws` with setting the `ROS_DISTRO='noetic'` before running the `stretch_create_ament_worskpace.sh`. Because `ROS_DISTRO` is set to 'noetic' and not 'galactic' at this point, the Installation kept exiting while executing the mentioned if statement. For this reason, I thought of sourcing `galactic` before the if statement.